staging/src/k8s.io/apiserver/pkg/util/flowcontrol/debug/dump.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package debug
+
+import (
+	"context"
+	"time"
+)
+
+// QueueSetDump is an instant dump of queue-set.
+type QueueSetDump struct {
+	Queues     []QueueDump
```

Comment:
For all `i` in [0, `len(Queues)`), `Queues[i].Index == i`, right?  If that's so then we do not need the `Index` field in a QueueDump.  If that's not so, then there is a surprise here.

Reply:
remove the `Index` field w/ the indices from arrays